src/group/tests/kat_transcripts.rs

```diff
@@ -190,13 +203,23 @@ pub fn run_test_vector(test_vector: TranscriptTestVector) -> Result<(), Transcri
         return Err(TranscriptTestVectorError::MembershipTagVerificationError);
     }
 
-    let my_confirmation_tag = confirmation_key.tag(&confirmed_transcript_hash_before);
+    //let my_confirmation_tag = confirmation_key.tag(&confirmed_transcript_hash_before);
+    let confirmed_transcript_hash_after =
+        hex_to_bytes(&test_vector.confirmed_transcript_hash_after);
```

Comment:
This seems to trigger a clippy warning above (L181), because `confirmed_transcript_hash_before` doesn't need to be cloned anymore.